code/game/objects/items/plushes.dm

```diff
@@ -599,6 +599,28 @@
 	icon_state = "goat"
 	desc = "Despite its cuddly appearance and plush nature, it will beat you up all the same. Goats never change."
 	squeak_override = list('sound/weapons/punch1.ogg'=1)
+	var/going_hard = FALSE
+
+/obj/item/toy/plush/goatplushie/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/clothing/mask/cigarette/rollie))
+		var/obj/item/clothing/mask/cigarette/rollie/fat_dart = I
+		if(!going_hard)
```

Comment:
invert this check, and if it's already going hard then early return.

example:
```dm
if(going_hard)
    to_chat(user, span_notice("[src] is already going too hard!"))
    return
```